Server/DeviceManagement/DeviceManagement.Domain/UserAggregate/Events/UserDeletedEvent.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System;
+using ModularHouse.Libraries.InternalMessaging.DomainEvents.Abstractions;
+
+namespace ModularHouse.Server.DeviceManagement.Domain.UserAggregate.Events;
+
+public record UserDeletedEvent(Guid TransactionId) : IDomainEvent;
```

Comment:
It makes sense to send some data about deleted `User`; Maybe `UserDto` or `Guid UserId`